apps/ima/views/summary.html

```diff
@@ -1,3 +1,4 @@
+<title>{{#t}}pages.summary.summary{{/t}} – GOV.UK</title>
 {{<partials-page}}
   {{$page-content}}
     <table class="table-details">
```

Comment:
I think you should remove this file completely and use the confirm.html file instead. You can add the template to the step in the ima/index.js like this:

> behaviours: [Summary, SaveFormSession],
> sections: require('./sections/summary-data-sections'),
> locals: { showSaveAndExit: true },
> template: 'confirm',

This is because the summary page at the end of the form will have to change it's path name from '/summary' to '/final-summary' and the summary page at the beginning of the form is already using a summary html file which has different information in it.

Reply:
Removed